Client/Client.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-      <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+      <TargetFrameworks>netstandard2.0;netstandard2.1;net5.0</TargetFrameworks>
```

Comment:
Please keep only the `netstandard` frameworks as targets.

Reply:
I think we should be adding 1 additional target due to it simplifying dependencies for everyone.

Reply:
@bednar the whole benefit of this PR is to exteralise the need for System.Collections.Immutable which can be achieved by adding net 5.